cloudify_cli/bootstrap/bootstrap.py

```diff
@@ -281,17 +308,28 @@ def get_protocol(rest_port):
         agent_remote_key_path = _handle_agent_key_file(fabric_env,
                                                        manager_node)
 
-        rest_client = utils.get_rest_client(manager_ip, rest_port, protocol,
-                                            skip_version_check=True)
```

Comment:
I am not sure, it's a code Pawel introduced. Maybe he thought that because it's during bootstrap there's no need for it.